refactored code and updated tutorial text to reflect changes (#31050)
mdn/content: files/en-us/learn/tools_and_testing/cross_browser_testing/your_own_automation_environment/index.md

To start up a new test, you need to include the `selenium-webdriver` module like this:

```js

> const { Builder, By, until } = require("selenium-webdriver");

Reviewer: You need `Browser` here too, or the following reference to `Browser.FIREFOX` would fail. However you don't need `until` in this section, because you don't use it.

```suggestion
const { Builder, Browser.FIREFOX, By} = require("selenium-webdriver");
```

```

> Next, you need to create a new instance of a driver, using the `new webdriver.Builder()` constructor. This needs to have the `forBrowser()` method chained onto it to specify what browser you want to test with this builder, and the `build()` method to actually build it (see the [Builder class reference](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_Builder.html) for detailed information on these features).

Reviewer: ```suggestion
Next, you need to create a new instance of a driver, using the imported `Builder()` constructor. This needs to have the `forBrowser()` method chained onto it to specify what browser you want to test with this builder (browser options are defined in the imported `Browser` interface).
The `build()` method is chained at the end to actually builds the driver instance (see the [Builder class reference](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_Builder.html) for detailed information on these features).
```

Let's create a new test to allow us to explore this code as we talk about it. Inside your selenium test project directory, create a new file called `quick_test.js`, and add the following code to it:

```js

> const { Builder, Browser } = require("selenium-webdriver");

Reviewer: This example lasts several sections. We will use `By` so add it now.

```suggestion
const { Builder, Browser, By } = require("selenium-webdriver");
```

Reviewer: Actually no we dont!

But it is better to use a remote server location so the code is more flexible — when you start using a remote server to run your tests (see later on), your code will break if you try to use local paths.

Update your `example()` function as follows:

```js

> const { Builder, Browser } = require("selenium-webdriver");

Reviewer: We need `By` here, 
```suggestion
const { Builder, Browser, By } = require("selenium-webdriver");
```

> One of the most useful ways to find an element by CSS — the By.css method allows you to select an element using a CSS selector

Reviewer: ```suggestion
One of the most useful ways to find an element by CSS — the `By.css()` method allows you to select an element using a CSS selector
```

});
```

Add this to the bottom of the `example()` function now as shown below:

```js
const { Builder, Browser, By } = require("selenium-webdriver");

(async function example() {
  const driver = await new Builder().forBrowser(Browser.FIREFOX).build();

  driver.get(

>     "https://mdn.github.io/learning-area/tools-testing/cross-browser-testing/accessibility/native-keyboard-accessibility.html"

Author: Is there something that I'm missing here?  There doesn't seem to be a need for this suggestion.

>     "https://mdn.github.io/learning-area/tools-testing/cross-browser-testing/accessibility/native-keyboard-accessibility.html"

Reviewer: Nothing I can see. Let's ignore.

You should see the button's text label reported inside the console.

Let's do something a bit more useful. Replace the previous code entry with this line of code, `button.click();` as shown below:

```js
const { Builder, Browser, By } = require("selenium-webdriver");

(async function example() {
  const driver = await new Builder().forBrowser(Browser.FIREFOX).build();
  
  driver.get(

>     "https://mdn.github.io/learning-area/tools-testing/cross-browser-testing/accessibility/native-keyboard-accessibility.html"

Reviewer: @ShamarYarde It's a bit annoying, but it isn't wrong. A JavaScript method can take additional values as arguments and they'll just be dropped. The linter requires this and easier not to fight it.

2. Give it the following contents:

   ```js

>    const { Builder, By, Key } = require("selenium-webdriver");

Reviewer: Do we need `Key`?

Author: `Key` is used in this line: `await driver.findElement(By.name("q")).sendKeys("webdriver", Key.RETURN);`.
